src/route/index.ts

```diff
@@ -55,6 +57,15 @@ export function initApiServer(
         },
       });
 
+      instance.route({
+        method: "GET",
+        url: "/metrics",
+        handler: async (_request, reply) => {
+          reply.header("Content-Type", register.contentType);
+          register.metrics().then((data) => reply.code(200).send(data));
+        },
+      });
```

Comment:
Which port will metrics be exposed on?

Reply:
It's 8080 right now

Reply:
I this separate from the main application port? If you use the same port it means that metrics might get exposed on the internet.
It's often useful expose metrics on a separate port

Reply:
oh yeah good point, it is on the same port as the app so I'll update that to a metrics specific port. Thank you!